assets/react-hooks/use-wp-admin-fullscreen.js

```diff
@@ -1,5 +1,12 @@
 import { useLayoutEffect } from '@wordpress/element';
 
+const toggleGlobalStyles = ( enabled, classes ) => {
+	if ( enabled ) document.body.classList.add( ...classes );
```

Comment:
We should [avoid single line blocks](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#blocks-and-curly-braces).